### DISCUSSION TITLE
Backbone client compilation

### DISCUSSION THREAD

**Discussion by zssvaidar** [atrocore/atrocore#519] [Q&A] [answered]
I have problem of custom module front-end changes not applying on my page.
How is it possible to re-compile app.js with backbone code to comile all javascipt files to app.js?

Thank you in advance

**Comment by rratsun** [comment, marked as answer]
This is not provided. You can only develop module scripts, like [here](https://github.com/atrocore/atropim/tree/master/client/modules/pim).
While developing, you should copy all client (front-end) code from your module to the root client directory. Same action the system do when updates.
